entity-service/src/main/kotlin/com/egm/stellio/entity/service/EntitiesEventService.kt

```diff
@@ -0,0 +1,36 @@
+package com.egm.stellio.entity.service
+
+import com.egm.stellio.shared.model.EntitiesEvent
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.databind.SerializationFeature
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import org.slf4j.LoggerFactory
+import org.springframework.cloud.stream.binding.BinderAwareChannelResolver
+import org.springframework.messaging.MessageHeaders
+import org.springframework.messaging.support.MessageBuilder
+import org.springframework.scheduling.annotation.Async
+import org.springframework.stereotype.Component
+
+@Component
+class EntitiesEventService(
+    private val resolver: BinderAwareChannelResolver
+) {
+    private val logger = LoggerFactory.getLogger(javaClass)
+    private val mapper =
+        jacksonObjectMapper().findAndRegisterModules().disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
+            .setSerializationInclusion(JsonInclude.Include.NON_NULL)
+
+    @Async
+    // only keep the event in the method constructor, re-add entityType to EntitiesEvent model ?
```

Comment:
Can we discuss this point ?

Reply:
I think  entityType should be a property of EntityEvent